ParkingLot/ParkingBoy.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ParkingLot
+{
+    public class ParkingBoy
+    {
+        private string name;
+        private Dictionary<string, ParkingTicket> providedParkingTickets = new Dictionary<string, ParkingTicket>();
+        public ParkingBoy(string inputName, ParkingLot parkingLot1, ParkingLot parkingLot2)
+        {
+            this.name = inputName;
+            ManagedParkingLots.Add(parkingLot1);
+            ManagedParkingLots.Add(parkingLot2);
+        }
```

Comment:
maybe the constructor need to be changed after clarifying the requirements

Reply:
yes! good idea!